bears/natural_language/LanguageToolBear.py

```diff
@@ -53,8 +55,12 @@ def run(self,
         # language_check being there.
         from language_check import LanguageTool, correct
 
+        for requirement in list(self.__class__.REQUIREMENTS):
```

Comment:
To make this a little less ugly, I have started https://gitlab.com/coala/package_manager/merge_requests/91 .
We would use it like `REQUIREMENTS = AddressableRequirementSet([PythonImportRequirement(...), ...])`
Let me know what you think.  It isnt polished yet.

Reply:
coala core is using depman master, so this should now be ready to roll.